scripts/globals/bcnm.lua

```diff
@@ -573,6 +573,7 @@ function checkReqs(player, npc, bfid, registrant)
         [1090] = function() return ( player:hasKeyItem(tpz.ki.TOGGLE_SWITCH)                                                                                                ) end, -- Quest: Puppetmaster Blues
         [1091] = function() return ( mjob == tpz.job.COR and mlvl >= 66                                                                                                     ) end, -- Quest: Breaking the Bonds of Fate (COR LB5)
         [1092] = function() return ( toau == mi.toau.LEGACY_OF_THE_LOST                                                                                                     ) end, -- TOAU35: Legacy of the Lost
+        [1122] = function() return ( player:getQuestStatus(AHT_URHGAN,tpz.quest.id.ahtUrhgan.OMENS) == QUEST_ACCEPTED and player:getCharVar('BluAf2CS') == 1                ) end, -- Quest: Omens (BLU AF2)
```

Comment:
Can we change the charger to the quest name. Quest_Omens for all scripts? Thoughts?

Reply:
Charger? what does that mean? ^^;;

Reply:
Sorry. Phone and auto correct. 
CharVar is what that was meant to be

Reply:
Sure we can, that will help with readability :cat:
I'll change it today

Reply:
resolved